package.json

```diff
@@ -59,6 +59,7 @@
     "thelounge-ldapjs-non-maintained-fork": "1.0.2",
     "ua-parser-js": "0.7.17",
     "urijs": "1.19.1",
+    "uuid": "3.1.0",
```

Comment:
This package is now in [v3.2.1](https://github.com/kelektiv/node-uuid/blob/master/CHANGELOG.md#320-2018-01-16).